### ISSUE TITLE
islandcraft not in multiverse generators list- generator not found- 1.12 

### ISSUE THREAD

**Issue by BluBeardedYeti** [hoqhuuep/IslandCraft#23]
Hello, 
After updating spigot to 1.12 from 1.11.2 IslandCraft is no longer recognized as a world gen by Multiverse. Both the Islandcraft plugin and the Multiverse plugin are updated to the latest version as well. I show no errors in the console so im at a loss.

multiverse core ver 2.5.0
islandcraft ver 1.0.8

**Comment by BluBeardedYeti** [comment]
multiverse core ver 2.5.0
islandcraft ver 1.0.8

**Comment by BluBeardedYeti** [comment]
update I did see the error 
Error:	ebean.properties not found
Times Occurred:	1
Plugin/Part Name:	Couldn't determine plugin name
Error Time::	[18:06:20]
What we suggest::	Couldn't determine help needed to solve this error, maybe try reinstalling the plugin?
Full Stacktrace::	
ebean.properties not found

I noticed on the main git page where you had mentioned updating to 1.12 and wanting to move away from this so i did a test and it seems it was the islandcraft plugin throwing that error.

I disabled ver 1.0.8 and installed ver 1.0.7 this error went away and the warning came up 'need to update islandcraft' but no ebean error.

uninstalled ver 1.0.7 and reinstalled 1.0.8 and got the- Error ebean.properties not found- again

hope this helps to get it sorted :)

**Comment by BluBeardedYeti** [comment]
Ok i went ahead and did some researching and found a solution. 

I created a blank ebean.properties.txt file in the folder where the server.properties file is, restarted the server, and the error was no longer in the console.

After this I was finally able to use multiverse to create the worlds I had set up in the Islandcraft config. 

yay XD sorry it wasnt my intention to use this forum for thinking out loud I just dont know how to wait for answers haha i just keep trying to work it out. But if anyone else is having this issue maybe this will help.
